src/pluto/helpers.py

```diff
@@ -73,9 +78,8 @@ def get_data(branch) -> Dict[str, pd.DataFrame]:
 
 def get_branches():
```

Comment:
Imagined this would also be moved to the digital ocean client. We could perform the blacklist logic on the list that is returned by the client, that seems like a fine design

Reply:
you're saying basically that lines 80-83 should be in digital_ocean_client? I limited how much I moved because i didn't want to end up with any like "if repo_name == 'pluto' do this kind of logic, but i think what you're making makes sense and should be fine

Reply:
This is a great point, I wasn't thinking about how disparately organized the data product folders are. I think they all should be organized like pluto and maybe when that happens we can give this method the name `get_all_branch_folders`.
The improvement is exactly what I had in mind